config/initializers/zxcvbn_overrides.rb

```diff
@@ -1,12 +1,14 @@
 require 'zxcvbn'
 
-module Zxcvbn
-  class Tester
-    attr_accessor :data_path
-    def initialize
-      @data_path = Rails.root.join('node_modules', 'zxcvbn', 'dist', 'zxcvbn.js')
-      src = File.open(@data_path || DATA_PATH, 'r').read
-      @context = ExecJS.compile(src)
+if Identity::Hostdata.instance_role != 'worker'
```

Comment:
I tried to get a Rails console on a worker box and this barfed, so I figured we just remove this initializer from workers, it shouldn't be needed